docs/geedocs/5.2.4/answer/7160003.html

```diff
@@ -0,0 +1,438 @@
+<div class="sidebar1">
+  <p class="sidebar-toc"><a href="../answer/7160004.html">Open GEE 5.2.4</a></p>
+  <p class="sidebar-toc"><a href="../answer/7160003.html" class="current-file">Open GEE 5.2.3</a></p>
+  <p class="sidebar-toc"><a href="../answer/7160002.html">Open GEE 5.2.2</a></p>
+  <p class="sidebar-toc"><a href="../answer/7160001.html">Open GEE 5.2.1</a></p>
+  <p class="sidebar-toc"><a href="../answer/7160000.html">Open GEE 5.2.0</a></p>
+  <p class="sidebar-toc"><a href="../answer/7159245.html">GEE 5.1.3</a></p>
+  <p class="sidebar-toc"><a href="../answer/6259403.html">GEE 5.1.2</a></p>
+  <p class="sidebar-toc"><a href="../answer/6159443.html">GEE 5.1.1</a></p>
+  <p class="sidebar-toc"><a href="../answer/6078762.html">GEE 5.1.0</a></p>
+  <p class="sidebar-toc"><a href="../answer/6053170.html">GEE 5.0.2</a></p>
+  <p class="sidebar-toc"><a href="../answer/4556704.html">GEE 5.0.1</a></p>
+  <p class="sidebar-toc"><a href="../answer/3424202.html">GEE 5.0</a></p>
+</div>
+<div class="content">
+<a name="top_of_file"></a>
+<p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
+<h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Release notes</h1>
+<h2>Release notes: Open GEE 5.2.3</h2>
+
+<p>Open GEE 5.2.3 is currently in development</p>
+
+  <h5>
+    <p id="overview_5.2.3">New Features</p>
+  </h5>
+    <p><strong>C++11 support</strong>. With release 5.2.3 Open GEE C++ code now supports and defaults to compiling using the <code>-std=gnu++11</code> compile flag.  This default can be overridden and the C++ code can be compiled using the <code>-std=gnu++98</code> flag instead.  However, Open GEE 5.2.3 will be the last release to support compiling C++ code using the <code>-std=gnu++98</code> flag.  The subsequent releases after Open GEE 5.2.3 will require the use of the <code>-std=gnu++11</code> flag when compiling C++ code.</p>
+    <p><strong>Remove TCP/IP connections to PostgreSQL</strong>. With release 5.2.3 Open GEE code will now connect to PostgreSQL exclusively using Unix domain sockets. This will make it easier to secure PostgreSQL for security sensitive installations </p>
+    <p><strong>Volume deletion</strong>. <code>geconfigureassetroot</code> now supports the option <code>--removevolume</code> for volume deletion.</p>
+    <p><strong>Performance tuning for gecombineterrain</strong>. <code>gecombineterrain</code> now supports the option <code>--numCompressThreads</code> which allows configuration of the threads used for operation.  If not specified <code>gecombineterrain</code> defaults this option to match the number of available CPUs.</p>
+    <p><strong>Package name customization at build time</strong>. Scons build now accepts a custom label argument which is appended to the version in RPM file names.</p>
+    <p><strong>Rewrite KML URLs</strong>. Users can rewrite the URLs for KML resorces included in databases at publish time.</p>
```

Comment:
I think a line was removed here that was in the 5.2.3 directory version of this file.  The line in my diff shows as:
<     <p><strong>Maps API Javascript Files</strong>. Maps API Javascript files are now available and installed at
 <code>/opt/google/gehttpd/htdocs/maps/mapfiles</code>.</p>

Reply:
Good catches.  They should be fixed now.